server/controllers/groups/create.js

```diff
@@ -29,7 +33,8 @@ module.exports = (req, res) => {
       description: description || '',
       searchable,
       position: position || null,
-      creatorId: req.user._id
+      creatorId: req.user._id,
+      open: open || false
```

Comment:
```suggestion
      open
```
There should be no need to re-parse the boolean value, `sanitize` is already doing it. It could be worth making explicit the boolean default default value (`false`), by setting `default: false` in the `sanitization` object (but then do it too for the `searchable` flag)